### ISSUE TITLE
Lich King encounter in mythic mode

### ISSUE THREAD

**Issue by Greenflamer** [dalaranwow/dalaran-wow#5820]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**:
Whenever you disconnect from the game while fighting LK, you are unable to rejoin the fight.
You are outside in icecrown or even ported to your race startingzone as mentioned in another post.
https://github.com/dalaranwow/dalaran-wow/issues/4986

**Expected behaviour**:
As on every (or atleast most) other bosses you should be able to rejoin the fight.

**Steps to reproduce the problem**:
1. Fight LK
2. Disconnect with a method of your choice (try ALT+F4)
3. Log back in -> you are outside of ICC now

**Include proofs for this behaviour**
Not needed...
We all know it happens although its not supposed to be like this.
Its actually a rather sloppy fix for an exploit the following exploit:
When valks grabbed a player he could ALT+F4 and log back in the game.
-> Valks never had to be killed and also raid never had to be stacked.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Icecrown Citadel - 4812
Lich King - 36597

**Include Screenshots from the issue if necessary**
Instead of a screenshot i wanna add the note that i totally agree on fixing the original exploit!
But not like this. The amount of fucked attempts because of this issue is way to high!
Please find another way to keep the exploit fixed but without increasing difficulty to mythic.

**Comment by STdimarx** [comment]
I agree that its ridiculous. 15 mins fight and any kind of disconnect for whatever reasons ruins the whole attempt.

**Comment by rakarudi** [comment]
This happens to us every week, always someone that DC on LK25 HC and when they come back online they are outside of instance and can not keep fight up.

Please prio this issue.
